File: PRACTICAS/practica3/Civitas_P1/src/civitas_p1/OperacionInmobiliaria.java

package civitas_p1;

public class OperacionInmobiliaria {
    
    private int numPropiedad;
    private GestionesInmobiliarias gestion;
    
    public OperacionInmobiliaria(GestionesInmobiliarias gest,int ip){
        
        gestion=gest;
        numPropiedad=ip; 
        
    }
    public GestionesInmobiliarias getGestion(){
        
        return gestion;
                
    }
    public int getNumPropiedad(){
        
        return numPropiedad;
                
    }
    
}

File: PRACTICAS/CAMBIAR_RUBY/RUBY2/Civitas/pkg/RubyApplication1-0.0.1/lib/operacion_inmobiliaria.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

module Civitas
  class OperacionInmobiliaria
    attr_reader :gestion, :numPropiedad
    def initialize (gest, ip)
      @gestion = gest
      @numPropiedad = ip
    end
  end
end
